scripts/gen_cli_help_screenshots.py

```diff
@@ -0,0 +1,46 @@
+import os
+import subprocess
+from pathlib import Path
+
+from rich.console import Console
+
+from commitizen.cli import data
+
+project_root = Path(__file__).parent.parent.absolute()
+images_root = project_root / Path("docs") / Path("images") / Path("cli_help")
+
+
+def gen_cli_help_screenshots() -> None:
+    """Generate the screenshot for help message on each cli command and save them as svg files."""
+    if not os.path.exists(images_root):
+        os.makedirs(images_root)
+        print(f'created {images_root}')
```

Comment:
```suggestion
        print(f"Created {images_root}")
```

Reply:
we can run ruff or black against these files to improve style consistency

Reply:
Okay. I run `ruff fomat` on this file and its style is  consistent now.
Btw, I notice only `commitizen` and `tests` are included  in the `scripts/format` as below.
Do you think including scripts and other folders is a good idea ?

<img width="446" alt="image" src="https://github.com/commitizen-tools/commitizen/assets/60340631/dcabb17b-9c70-4915-86a1-17cd55901c27">

Reply:
yep, I think that what we should do